src/test/ui/repr/repr-transparent.stderr

```diff
@@ -21,31 +21,43 @@ LL | pub struct StructWithProjection(f32, <f32 as Mirror>::It);
 error[E0691]: zero-sized field in transparent struct has alignment larger than 1
   --> $DIR/repr-transparent.rs:36:32
    |
+LL | #[repr(transparent)]
+   |        ----------- help: Try using `repr(C)` instead
 LL | struct NontrivialAlignZst(u32, [u16; 0]);
-   |                                ^^^^^^^^ has alignment larger than 1
+   |                                ^^^^^^^^ has alignment of 2, which is larger than 1
 
 error[E0691]: zero-sized field in transparent struct has alignment larger than 1
   --> $DIR/repr-transparent.rs:42:24
    |
+LL | #[repr(transparent)]
+   |        ----------- help: Try using `repr(C)` instead
 LL | struct GenericAlign<T>(ZstAlign32<T>, u32);
-   |                        ^^^^^^^^^^^^^ has alignment larger than 1
+   |                        ^^^^^^^^^^^^^ has alignment of 32, which is larger than 1
+
+error[E0691]: zero-sized field in transparent struct has alignment larger than 1
+  --> $DIR/repr-transparent.rs:45:33
+   |
+LL | #[repr(transparent)]
+   |        ----------- help: Try using `repr(C)` instead
+LL | struct GenericAlignZeroArray<T>([T; 0], u32);
+   |                                 ^^^^^^ may have alignment larger than 1
```

Comment:
also interested in why the number of errors has increased

Reply:
edit: d'oh, actually it's because you edited the UI test. my bad. resolving this comment.